src/colorizer/AnnotationData.ts

```diff
@@ -10,18 +10,32 @@ const CSV_COL_ID = "ID";
 const CSV_COL_TIME = "Frame";
 const CSV_COL_TRACK = "Track";
 
+export const BOOLEAN_VALUE = "TRUE";
```

Comment:
Nit: how is this value written to csv?  it would be good if it were something easily parseable by json or Python like `true/false`, `True/False`, or `1/0`
Also I'm not sure whether you need an explicit "false" state to write to csv.  I realize that for other value types you need an empty "not set" kind of value and empty string fits the bill.

Reply:
Ah, yeah, I can do lowercase `true`. I'll ask Chantelle/Serge if an empty string is okay for boolean values in place of `false`, but it's easy to do either.